app/src/main/java/org/simple/clinic/util/LocaleOverrideContextWrapper.kt

```diff
@@ -0,0 +1,26 @@
+package org.simple.clinic.util
+
+import android.content.Context
+import android.content.ContextWrapper
+import java.util.Locale
+
+class LocaleOverrideContextWrapper(base: Context) : ContextWrapper(base) {
```

Comment:
This constructor is only being used in this class, let's make it `private constructor`?